manticore/ethereum/manticore.py

```diff
@@ -1019,22 +1036,24 @@ def multi_tx_analysis(
         args=None,
         compile_args=None,
     ):
-        owner_account = self.create_account(balance=10000000000000000000, name="owner")
-        attacker_account = self.create_account(balance=10000000000000000000, name="attacker")
+        owner_account = self.create_account(balance=10**10, name="owner")
+        attacker_account = self.create_account(balance=10**10, name="attacker")
         # Pretty print
         logger.info("Starting symbolic create contract")
 
         if tx_send_ether:
             create_value = self.make_symbolic_value()
         else:
             create_value = 0
+
         contract_account = self.solidity_create_contract(
             solidity_filename,
             contract_name=contract_name,
             owner=owner_account,
             args=args,
             compile_args=compile_args,
             balance=create_value,
+            gas=230000,
```

Comment:
2300?
 
Same question applies to line 1099 below.

Reply:
It got important now. Because the gas got debited from the server and all that.
I just wanted to give a generic amount that is normally enough to run through the creation. 
Maybe we need a conf?  idk